conans/client/generators/cmake_common.py

```diff
@@ -168,22 +168,27 @@ def cmake_global_vars(deps, build_type=""):
     # Property INTERFACE_LINK_FLAGS do not work, necessary to add to INTERFACE_LINK_LIBRARIES
     set_property(TARGET {name} PROPERTY INTERFACE_LINK_LIBRARIES ${{CONAN_PACKAGE_TARGETS_{uname}}} ${{_CONAN_PKG_LIBS_{uname}_DEPENDENCIES}}
                                                                  $<$<STREQUAL:$<TARGET_PROPERTY:TYPE>,SHARED_LIBRARY>:${{CONAN_SHARED_LINKER_FLAGS_{uname}_LIST}}>
+                                                                 $<$<STREQUAL:$<TARGET_PROPERTY:TYPE>,MODULE>:${{CONAN_SHARED_LINKER_FLAGS_{uname}_LIST}}>
```

Comment:
`MODULE_LIBRARY`, not `MODULE`. Sorry for not mentioning that correctly in bug description (I've updated it).